src/plugins/vcDoubleClick/index.ts

```diff
@@ -52,7 +52,7 @@ export default definePlugin({
             replacement: {
                 match: /onClick:(\i)(?=,.{0,30}className:"channelMention".+?(\i)\.inContent)/,
                 replace: (_, onClick, props) => ""
-                    + `onClick:(vcDoubleClickEvt)=>$self.shouldRunOnClick(vcDoubleClickEvt,${props})&&${onClick}()`,
+                    + `onClick:(vcDoubleClickEvt)=>$self.shouldRunOnClick(vcDoubleClickEvt,${props})(vcDoubleClickEvt.stopPropagation(),${onClick}())`,
```

Comment:
this looks wrong, shouldn't it be this?
```suggestion
                    + `onClick:(vcDoubleClickEvt)=>$self.shouldRunOnClick(vcDoubleClickEvt,${props})&&(vcDoubleClickEvt.stopPropagation(),${onClick}())`,
```

Reply:
also it seems the `find` for this patch is no longer unique and needs updating